apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -357,12 +365,48 @@ export const PoolTable = ({ pools }: PoolTableProps) => {
 };
 
 const PoolTableApi = () => {
-  const [result] = useGetPoolsQuery();
+  // data manipulation functions
```

Comment:
These two functions could be moved outside the component so they don't get recreated on every render?